lib/locales/en/overwatch.yml

```
@@ -37,5 +37,332 @@ en:
           "Wait for it...",
           "Warīhum quwitak!",
           "You're powered up, get in there.",
-          "¡Apagando las luces!"
+          "¡Apagando las luces!",
+          "A steady blade balances the soul.",
+          "Come on!",
+          "Kuso!",
+          "Kakugo!",
+          "Iza jinjō ni shōbu!",
+          "Measure twice, cut once.",
+          "Waga tamashī wa kinkō o motomeru.",
+          "Mada mada!",
+          "Hah! Simple.",
+          "Yoshi!",
+          "You are only human.",
+          "I was hoping for a challenge.",
+          "My Halloween costume? Cyborg ninja.",
+          "Merry Christmas!",
+          "To know yourself is to be at peace.",
+          "Watch and learn.",
+          "You seem familiar. Ain’t I killed you before?",
+          "Happens to the best of us.",
+          "I’m the quick. You’re the dead.",
+          "I’m your Huckleberry.",
+          "I’ve got a bullet with your name on it.",
+          "I tried being reasonable, didn’t take to it.",
+          "Reach for the sky.",
+          "I’m not good. Not bad. But I sure as hell ain’t ugly.",
+          "Wanted: Dead or alive.",
+          "You done?",
+          "I don’t much like losin’.",
+          "It’s your funeral.",
+          "This calls for a celebration.",
+          "Put your security in my hands.",
+          "Aerial superiority achieved.",
+          "Flying the friendly skies.",
+          "I’ve got you on my radar.",
+          "Leave this to a professional.",
+          "Not a chance.",
+          "Play nice, play Pharah.",
+          "Rocket jump? That sounds dangerous.",
+          "Shot down.",
+          "Sorry, but I need to jet.",
+          "We are in this together.",
+          "You’re going to come with me, dead or alive.",
+          "Want to know the forecast?",
+          "I keep the peace.",
+          "What are you looking at?",
+          "Dead man walking.",
+          "Give me a break.",
+          "Haven’t I killed you somewhere before.",
+          "I’m back in black.",
+          "If it lives, I can kill it.",
+          "Next.",
+          "I’m not a psychopath, I’m a high-functioning psychopath.",
+          "You look like you’ve seen a ghost.",
+          "Too easy.",
+          "Was that all?",
+          "It’s in the refrigerator.",
+          "I work the graveyard shift.",
+          "Where’s your holiday spirit?",
+          "Now those are some fireworks.",
+          "I’ve still got it.",
+          "Young punks… Get off my lawn!",
+          "I’m an army of one.",
+          "I didn’t start this war… But I’m damn well gonna finish it.",
+          "Not on my watch.",
+          "Old soldiers never die… And they don’t fade away…",
+          "Smells like victory…",
+          "That’s "Sir",
```

Hmm, this is causing the build to fail.
Maybe you meant
```suggestion
          "That’s \"Sir\" to you",
```
?